Docs 8010 edit api groups pa (#255)
mulesoft/docs-api-manager: modules/ROOT/pages/api-groups-creating-groups.adoc

> = Create API Groups and Add API Instances

Reviewer: Superfast proof/scan review

Author: ok, Thanks!

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> You can bundle a set of APIs to enable ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.

Reviewer: ```suggestion
You can bundle a set of APIs to increase ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.
```

Author: Removed that para and added this:
o package a set of APIs, you first create an API Group and then add the relevant API instances to the group. If you want your API Group to be discovered by consumers, you must publish it to Exchange.

> When you complete the task of creating an API Group with the help of this topic, you would have created an API Group, the very first version of that group, and the very first group instance. Additionally, you would also have added API instances to the API Group instance.

Reviewer: ```suggestion
Using the following information, you create an API Group, including creating an API Group, creating the very first version of that group, and creating the very first group instance. Additionally, you add API instances to the API Group instance.
```

Author: Done! Thanks!

> You can add API instances to the API Group instance based on the environment (Production, Sandbox, or others) in which you create the API Group instance. For example, if you created a API Group instance in Production environment, you can add API instances to the API Group instance only from Production.

Reviewer: ```suggestion
You can add API instances to the API Group instance based on the environment (production, sandbox, or others) in which you create the API Group instance. For example, if you created an API Group instance in a production environment, you can add API instances to the API Group instance only from production.
```
GLOBALLY lowercase environment names, per Term Presentation List.

Author: Done! Thanks!

You create API Groups using the following steps:

> 

Reviewer: Do we really need a list for just two tasks, and why do we not number them instead of boldfacing them?

Author: Numbered the tasks, but need them to clarify that you create a group in two stages.

*Step 1:* <<create-api-groups,Create an API Group>>

> The first step to creating an API Group that can be used by consumers, is create an API Group. 

Reviewer: ```suggestion
The first step to creating an API Group that can be used by consumers is to create an API Group. 
```

Author: Done! Thanks!

*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>

After you create the API Group, you then add API instances to this group for the first time.

== Task Prerequisites

Before you begin, you must have:

> * At least one API instance in your organization or business group.

Reviewer: ```suggestion
* At least one API instance in your organization or business group
```

Author: Done! Thanks!

// +
// Otherwise, you may not find the required organization or business group in the list.

> * API Group Administrator permissions to perform to create API Groups .

Reviewer: ```suggestion
* API Group Administrator permissions to create API Groups
```

Author: Done! Thanks!

//Group administrators is a permission which is set a business group *BG) level. A group administrator can manage (create, update, delete) all groups of the BG. The permissions algo, gives user VIEW access to all API instances of the BG (so user can add them to the group)

[[create-api-groups]]
== Create API Groups

You create an API Group at the organization level.

To create API Groups:

. From the left menu in API Manager, click *API Groups > Create API Groups*.
+

> The Create New API Group window appears.

Reviewer: Going forward, omit all these sentences about what appears. We assume the step worked and the following step applies to the result of the one before it.

Author: Ok Done! Thanks!

. In the *API Group Name* field, specify a name for the API Group that you want to create: for example, `Social Media`.
. In the *API Group version* field, specify the version for the API Group, for example, `v1`.

> . Optionally, in the API Group instance label field, specify a label for the API Group, such as `Internal`, if it is to be used for internal purposes.

Reviewer: passive

Author: Done! Thanks!

. Click *Ok*.
+
The *Creating a new group* page is displayed. You are now ready to <<adding-api-instances-to-groups,add API instances>> to the API Group.

[[adding-api-instances-to-groups]]

> == Add API instances to API Groups

Reviewer: ```suggestion
== Add API Instances to API Groups
```

Author: Done! Thanks!

> Your API Group is not complete and functional unless you add the API instances to the group that you want to bundle. When you add API instances to an API Group, you can add:

Reviewer: ```suggestion
Your API Group is not functional until you add the API instances to the group that you want to bundle. When you add API instances to an API Group, you can add:
```

Author: Done! Thanks!

> 

Reviewer: No periods; these are fragments

Author: Done! Thanks!

* Only those API instances that belong in the same business group or organizations.
* Only those API instances that belong the same environment as the API Group instance.

> * Only those API instances that belong to the same identity provider (IdP) as the API Group instance. You cannot mix IdPs in an API Group instance.

Reviewer: ```suggestion
* Only those API instances that belong to the same identity provider (IdP) as the API Group instance +
You cannot mix IdPs in an API Group instance.
```

Author: Done! Thanks!

To add a new API instance to a new API Group:

. On the *Creating a new group* page, select an environment from the *Select environment* drop-down list.
+
.The screen capture shows how to add API instance to an API Group.
image:create-api-instances-api-groups.png[Adding API instances to API Groups]

> +

Reviewer: NO PERIODS and format UI element labels.

Author: Done! Thanks!

<1> The organization under which the API Group is created.
<2> The Add button to add more API instances to the group.
<3> The Add Organization button to add organizations to the group.
<4> The Save button.
<5> The delete button to instantly delete an API instance from the group.

> . From the *Select an API* list, choose the API instance that you want to add.

Reviewer: ```suggestion
. From the *Select an API* list, select the API instance that you want to add.
```

Author: Done! Thanks!

> . From the *Select a version / instance* list, choose the version of the API Instance that you want to add to the Group.

Reviewer: ```suggestion
. From the *Select a version / instance* list, select the version of the API Instance that you want to add to the group.
```

Author: Done! Thanks!

. If you want to include additional API instances in the API Group, click *Add* and specify the details.
. If the API instance and the API Group belong in different business groups:
.. Click *Add Organization*.
.. Select the organization or business group in which your API instance is available.

> .. From the environment, select the API instance, and version that you want to add.

Reviewer: ```suggestion
.. From the environment, select the API instance and version that you want to add.
```

Author: Done! Thanks!

. Click *Save*.
+
Your API instances are now added to the API Group.

== See Also

* xref:api-groups-landing-page.adoc[API Groups]
* xref:api-groups-modifying-groups.adoc[Modify API Groups]
* xref:api-groups-publishing-to-exchange.adoc[Publishing API Groups to Exchange]
* xref:api-groups-deprecating-groups.adoc[Deprecate API Groups]

> * xref:api-groups-deleting-groups.adoc[Deleting API Groups]

Reviewer: ```suggestion
* xref:api-groups-deleting-groups.adoc[Delete API Groups]
```

Reviewer: BREAK TO SEND EDITS SO FAR

Reviewer: RESUME

Author: Done! Thanks!
